### PR TITLE
Include the training module

### PR DESCRIPTION
**Pull request by ajacks-msft**
Replacing the detailed steps with an overview step. Adding a link to the training module.

### CODE REVIEW

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/data-connect-get-started.md](https://github.com/ajacks-msft/microsoft-graph-docs/blob/patch-1/concepts/data-connect-get-started.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C8%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811080134557682-3356%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ICt%2BKGaxF9Kf9sGKynsTAqgQxaCtUUo1v5U9OoZarvc%3D&st=2018-11-08T01%3A31%3A53Z&se=2018-12-09T01%3A36%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/data-connect-get-started.md](https://github.com/ajacks-msft/microsoft-graph-docs/blob/patch-1/concepts/data-connect-get-started.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C8%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811082002426650-3356%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=GpZVXxMUvIszx8h62Qhn5AK99TDCaEhTd%2FCn2ZThFaU%3D&st=2018-11-08T19%3A59%3A28Z&se=2018-12-09T20%3A04%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/data-connect-get-started.md [hunk 1]
```diff
@@ -2,23 +2,9 @@
...
-5.	Enter an email address for the group (for example, approvers@contoso.com).
-6.	Choose **Add**.
-7.	Go to [Home > Services & add-ins](https://portal.office.com/adminportal/home#/Settings/ServicesAndAddIns) in the Microsoft 365 admin center.
-8.	Open **Microsoft Graph data connect preview**.
-9.	Flip the switch to **On**.
-10.	Under **Group of users to make approval decisions**, enter an alias for a mail-enabled security group (for example, approvers for approvers@contoso.com).
-11.	Choose **Save**.
-12.	Wait until the settings update confirmation message is displayed: Microsoft Graph data connect preview settings have been updated.
+1. Give consent to opt in to Microsoft Graph data connect through the Microsoft 365 Admin Portal, on the **Services & add-ins** page. This will allow data movement requests to Microsoft Azure (that is, keep full control over the data, but allow Azure resources to access it). No data is transferred unless approval for a specific pipeline is provided later.
+2. Set an approver group within the Office 365 subscription. The approver group will be tasked with approving specific requests for access to data.
```
Comment (tlenig) [inline concepts/data-connect-get-started.md]:
```suggestion
2. Set an approver group within the Office 365 subscription. Ensure the approver group is not empty, as only the users in the group can approve data movement requests. 
```

Adding a small comment inline, we received feedback from our customers that they didn't understand that the group needed to have users in it to be valid.

Comment (Lauragra) [reply concepts/data-connect-get-started.md]: Suggested edit to Tyler's addition: Make sure that the approver group is not empty. Only the users in the group can approve data movement requests.

File: concepts/data-connect-get-started.md [hunk 2]
```diff
@@ -2,23 +2,9 @@
...
 
 ## Next steps
-Congratulations! You are now ready to start building intelligent applications with Azure tooling. For a sample application and additional documentation, see the [Microsoft Graph data connect GitHub repo](https://github.com/OfficeDev/MS-Graph-Data-Connect/wiki). 
\ No newline at end of file
+
+Congratulations! You're now ready to start building intelligent applications with Azure tooling. For detailed step-by-step instructions, see the [Microsoft Graph data connect training module](https://github.com/microsoftgraph/msgraph-training-dataconnect/blob/master/Lab.md). For a sample application and additional documentation, see the [Microsoft Graph data connect GitHub repo](https://github.com/OfficeDev/MS-Graph-Data-Connect/wiki).
```
[the remark this answers is not in the document]
Comment (angelgolfer-ms) [reply concepts/data-connect-get-started.md]: Had a chat with @Lauragra - she's right that the getting started topic gets much shorter after eliminating the 12 steps. On the other hand, there's value in keeping the getting started topic in the TOC to make it discoverable. Not every new customer goes through the value prop overview topic (if they have heard data connect elsewhere), but every new customer will need to get set up to use data connect.

[the remark this answers is not in the document]
Comment (tlenig) [reply concepts/data-connect-get-started.md]: Great feedback Angela! I certainly agree with your first comment about moving the "For detailed step-by-step" sentence higher up. 

I'm also comfortable leaving the getting started topic in place for now, maybe we can merge it with the value prop topic if we get feedback on it being too short?

File: concepts/data-connect-get-started.md (hunk 1 above)
Comment (tlenig) [reply concepts/data-connect-get-started.md]: Looks good to me thanks Laura!

Comment (ajacks-msft) [reply concepts/data-connect-get-started.md]: Updated.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/data-connect-get-started.md](https://github.com/ajacks-msft/microsoft-graph-docs/blob/patch-1/concepts/data-connect-get-started.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C9%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811092050134129-3356%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=z4eqXke2JYjfOUbfOFV%2B5%2BIeyTSJhve0IZVpI8zn6OE%3D&st=2018-11-09T20%3A46%3A56Z&se=2018-12-10T20%3A51%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/data-connect-get-started.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ajacks-msft) [reply concepts/data-connect-get-started.md]: I've moved the line about the training module. If we decide to merge two of the pages, let's use another branch.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/data-connect-get-started.md](https://github.com/ajacks-msft/microsoft-graph-docs/blob/patch-1/concepts/data-connect-get-started.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C13%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811131829292436-3356%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=mL6ZNkjCER5aDRqFBVZTdc4LVnjIQihU7Inrc4VFeUc%3D&st=2018-11-13T18%3A26%3A48Z&se=2018-12-14T18%3A31%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/data-connect-get-started.md](https://github.com/ajacks-msft/microsoft-graph-docs/blob/patch-1/concepts/data-connect-get-started.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C13%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811131946356054-3356%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=rtYLSba3piOokaKvcPPUgJ6%2BwwZpocL%2BSasiCk%2BZSWw%3D&st=2018-11-13T19%3A43%3A41Z&se=2018-12-14T19%3A48%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/data-connect-get-started.md](https://github.com/ajacks-msft/microsoft-graph-docs/blob/patch-1/concepts/data-connect-get-started.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C14%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811142249154810-3356%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9W6GbXryy6HLdSNBnfZvXUYo4Q%2BQjl6Li2oddUe9cs4%3D&st=2018-11-14T22%3A46%3A27Z&se=2018-12-15T22%3A51%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**